charts/kargo/templates/external-webhooks-server/deployment.yaml

```diff
@@ -82,42 +82,43 @@ spec:
           {{- toYaml . | nindent 8 }}
         {{- end }}
         ports:
-        - containerPort: 9443
```

Comment:
This was copy pasta from the admission control webhooks server, which unconditionally uses TLS. Switching to 8080 makes the external webhooks server more consistent with the API server. The external webhook server actually resembles the API server much more than it does the admission control webhooks server.